.github/workflows/validate-json.yaml

```diff
@@ -0,0 +1,39 @@
+name: Validate JSONs
+
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+
+jobs:
+  validate-json:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install V8
+        run: sudo apt install libv8-dev
+
+      - name: Install jsonvalidate
+        run: |
+          Rscript -e 'install.packages("jsonvalidate")'
+
+      - name: Validate packages.json
+        run: |
+          jsonvalidate::json_validate(
+            json = jsonlite::read_json(path = "inst/extdata/data.json")[[1]],
+            schema = jsonlite::read_json(path = "inst/extdata/schema.json")[[1]],
+            engine = "ajv",
+            verbose = TRUE,
+            greedy = TRUE,
+            strict = TRUE,
+            error = TRUE
+          )
```

Comment:
I've spent a lot of time trying to figure out why this workflow fails as it does, and what alternatives there are. Can work on this further as well with a simpler example, but for now a few points:

1. The workflow fails as-is becasue some elements do not have the correct values for `prob_distribution`, these are rows 26:29, but checking those, it seems the values are more or less correct ("weibull" and NA), so unclear what's happening. The ones which are NA might be failing as the `prob_distr` field isn't present in the JSON.
2. When saving these in 'pretty' format and running the validator, it fails on checking whether the database is a JSON object; the error is not at all informative.

`{jsonvalidate}` requires a character string (or path) for both `json` and `schema`, and it's possible to get that by adding `jsonlite::toJSON()` around `jsonlite::read_json()`. That allows you to save the files as human-readable JSON, rather than the unreadable string currently used.

Reply:
I've updated the JSON files and functions that make them to be readable (`pretty = TRUE`), resolved in ffff318. 

For now I've removed the enum categories that were causing the issue by not properly rendering `NULL` from the yaml to the JSON. Perhaps in the future we can look into this.

Reply:
Thanks for taking a look - I'll also give the yaml -> JSON conversion a look to see if the enum problem can be resolved somehow.